README.md

```diff
@@ -154,20 +156,20 @@ promise2.then(function(result) {
 })
 ```
 
-## `td2.0-rest-connector`
+## `@tdengine/rest`
 
 This is a TDengine's RESTful connector in TypeScript. It's depend on [node-fetch v2](https://github.com/node-fetch/node-fetch/tree/2.x). Using `fetch(url,options)` to send sql statement and receive response.
 
 ### Installation
 
 ```bash
-npm i td2.0-rest-connector
+npm i @tdengine/rest
 ```
 
 ### Usage
 
 ```TypeScript
-import { options, connect } from 'td2.0-rest-connector'
+import { options, connect } from '@tdengine/rest'
 options.path='/rest/sqlt';
```

Comment:
in 3.0, taosadpater default support path "/rest/sql"
need to change to 
`options.path='/rest/sql'` or delete this line. since these is a default path